src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -28,6 +28,7 @@ internal class FunctionDispatcher : IFunctionDispatcher
         private readonly IDisposable _rpcChannelReadySubscriptions;
         private readonly int _debounceSeconds = 10;
         private readonly int _maxAllowedProcessCount = 10;
+        private readonly object _disposeLock = new object();
```

Comment:
what is this? doesn't appear to be used?

Reply:
thanks! missed deleting. Remved it.